docs/annotation_scripts.rst

```diff
@@ -0,0 +1,15 @@
+Batch annotation scripts
```

Comment:
Could we remove the "Batch" here. I don't think it's needed and we call these "Annotation Scripts" everywhere else.

Reply:
removed